### DISCUSSION TITLE
Is there a way to close all windows of vscode and restore them on next start

### DISCUSSION THREAD

**Discussion by hsupu** [microsoft/vscode-discussions#139] [Extension Show and Tell]
I know it works when I shutdown/restart my PC, but I couldn't find the way to do it manually. Could you help?

I found that there wasn't a proper category for my question..

**Comment by r3m0t** [comment]
https://github.com/microsoft/vscode-discussions/discussions/5

> Why are there just categories for extension development?
We want to start small, and the extension author community is just the right size. Meanwhile [StackOverflow](https://stackoverflow.com/questions/tagged/visual-studio-code?tab=Newest) continues to serve most other questions.

File > Exit works on Windows

There's also a related steting called "Files: Hot Exit".

**Reply by YakovL** [reply]
Do you mean there should be "Exit works on Windows" item in the "File" menu? I only have this:

![image](https://github.com/microsoft/vscode-discussions/assets/1131924/e972d9fd-da3c-489b-a9d2-fd9c03e7ced0)
